src/sentry/models/installation.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import absolute_import
+
+from django.db import models, IntegrityError, transaction
+
+from sentry.db.models import Model, FlexibleForeignKey
+
+
+class OrganizationInstallation(Model):
+    __core__ = False
+
+    organization = FlexibleForeignKey('sentry.Organization')
+    installation = FlexibleForeignKey('sentry.Installation')
+
+    class Meta:
+        app_label = 'sentry'
+        db_table = 'sentry_organizationinstallation'
+        unique_together = (('organization', 'installation'),)
+
+
+class Installation(Model):
```

Comment:
should we add some kind of config-like field to future proof this?

also im not sure this would work with bitbucket

e.g. id like to be able to say "this bitbucket authorization is associated with the org"

More or less, I think this should operate similar to UserSocialAuth/AuthIdentity, but instead of being coupled to a user, its coupled to an org. Additionally when possible it uses an org-wide auth flow (e.g. github integrations), and when not, uses a user-wide flow (e.g. bitbucket?)